### ISSUE TITLE
FCU BUTTON

### ISSUE THREAD

**Issue by DanP77** [headwindsim/A339X#67]
### Description

Has the EXP Button to be replaced by ALT Button?

**Comment by st3ph29** [comment]
No

**Comment by DanP77** [comment]
![image](https://github.com/headwindsim/A339X/assets/72685565/a627150d-63b4-4f1e-bd45-f40add11547b)

**Comment by st3ph29** [comment]
What has this to do with the issue section?

**Comment by DanP77** [comment]
> What has this to do with the issue section?

I just opened a "Feature Request"

**Comment by st3ph29** [comment]
Yes you open a "feature request" that doesn't sound as a request in an "Issue" tab. Better do it on the discord A330 family suggestion channel.

**Comment by DanP77** [comment]
> Yes you open a "feature request" that doesn't sound as a request in an "Issue" tab. Better do it on the discord A330 family suggestion channel.

It automatically open in this section.
And not everybody are able to use discord.
